app/_src/gateway-operator/get-started/kic/create-route.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Create a Route
+content-type: tutorial
+book: kgo-kic-get-started
+chapter: 3
+alpha: true
+---
+
+Now that you've installed all of the required components and configured a `GatewayClass`, it's time to route some traffic to a service in your Kubernetes cluster.
+
+## Configure the echo service
+
+In order to route a request using {{ site.base_gateway }} we need a service running in our cluster. Install an `echo` service using the following command:
+
+```bash
+kubectl apply -f {{site.links.web}}/assets/kubernetes-ingress-controller/examples/echo-service.yaml
+```
+
+## Create a HTTPRoute
+
+Now that we have a service to proxy requests to, we can create a `HTTPRoute` to send any requests that start with `/echo` to the echo service:
+
+```yaml
+echo '
+kind: HTTPRoute
+apiVersion: gateway.networking.k8s.io/v1beta1
+metadata:
+  name: echo
+spec:
+  parentRefs:
+    - group: gateway.networking.k8s.io
+      kind: Gateway
+      name: kong
+  rules:
+    - matches:
+        - path:
+            type: PathPrefix
+            value: /echo
+      backendRefs:
+        - name: echo
+          port: 1027
+' | kubectl apply -f -
+```
+
```

Comment:
This section needs to be indented.

Also needs the following lines:

The results should look like this:
```text
httproute.gateway.networking.k8s.io/echo created
```

Reply:
The indentation looks odd when there are only two steps and the rest of the page isn't indented. I've added the results but removed the indentation

Reply:
Actually I've reconsidered. Here's show it looks without a third item:

![CleanShot 2023-09-25 at 12 24 41](https://github.com/Kong/docs.konghq.com/assets/59130/a954fad6-72c8-45ba-af6c-1a35bdd13c6b)

But it looks ok with 3:

![CleanShot 2023-09-25 at 12 25 08](https://github.com/Kong/docs.konghq.com/assets/59130/5728ff79-29c1-445b-9072-197b692e5c7e)